pyanaconda/rescue.py

```diff
@@ -248,6 +246,11 @@ def _unlock_devices(self):
                                               parents=[device],
                                               exists=True)
                         self.storage.devicetree._add_device(luks_dev)
+
+                        # Wait for the device.
+                        # Otherwise, we could get a message about no Linux partitions.
+                        time.sleep(2)
```

Comment:
I wonder if there is something blocking we could call (udev settle/some blivet thing - I guess the storage guys might know) instead of arbitrary sleep ? Well, I can image there is nothing like that though.